examples/plot_meg_data_example.py

```diff
@@ -4,10 +4,12 @@
 
```

Comment:
can you add a paragraph here explaining what this example is about
and putting the ref to our paper?

Reply:
Done